crates/wasmtime/src/runtime/component/func/typed.rs

```diff
@@ -12,6 +12,7 @@ use core::marker;
 use core::mem::{self, MaybeUninit};
 use core::ptr::NonNull;
 use core::str;
+use std::iter;
```

Comment:
```suggestion
use core::iter;
```

That should fix https://github.com/bytecodealliance/wasmtime/actions/runs/14112166505/job/39533533593